src/huggingface_hub/hf_api.py

```diff
@@ -420,6 +420,8 @@ def unset_access_token():
     def list_models(
         self,
         filter: Union[str, Iterable[str], None] = None,
+        author: Optional[str] = None,
```

Comment:
I think this might be better named as `author_or_organization` instead of just author.

Reply:
just chiming in to say that on the hf.co hub side (and in APIs and the doc probably) we use `author` to mean `user_or_organization`

My 2 cents would be to keep `author` as a umbrella for both

Reply:
Sounds good with me then.